lib/errors.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const getContent = (err, debug, translate) => {
+  var content = {};
```

Comment:
if you're `const`ing and `let`ing perhaps stop `var`ing?

I think if we're gonna start forcing the es6 route we can stop with `var` altogether?